packages/jest-runtime/src/index.js

```diff
@@ -545,9 +546,28 @@ class Runtime {
       }
     }
 
-    const wrapper = this._environment.runScript(transformedFile.script)[
-      ScriptTransformer.EVAL_RESULT_VARIABLE
-    ];
+    const runScript = this._environment.runScript(transformedFile.script);
+
+    if (runScript === null) {
+      const originalStack = new ReferenceError(
```

Comment:
this is added in a second commit so easy to back out, not sure if it's needed. Without it you get this: 
![image](https://user-images.githubusercontent.com/1404810/38023931-d1a81e7c-3283-11e8-84c7-7143afb06fa7.png)

So not too bad either way 🙂